src/elements/content-sidebar/activity-feed/task-new/TaskDueDate.js

```diff
@@ -8,6 +8,9 @@ import { ReadableTime } from '../../../../components/time';
 
 import { TASK_NEW_NOT_STARTED } from '../../../../constants';
 
+import type { TaskStatus } from './flowTypes';
+import type { ISODate } from '../../../../common/box-types';
```

Comment:
probably rename file to flowTypes.js (or just add a new one and leave the old one for later refactor) so that it can just be common/flowTypes

Reply:
Done.

Reply:
We might end up with something like `flowTypes/tasks`, `/core`, `/miniObjects` etc. eventually but this is a good start - no more leaking globals